.github/pull_request_template.md

```diff
@@ -0,0 +1,20 @@
+## Description
+
+Please explain the changes you have made here.
+
+## Jira Ticket
+
+Please add a link to the related Jira ticket (if applicable).
+
+## Checklist
+
+These are items (excluding GitHub Checks) which should be confirmed before a branch is ready to merge.
+
+- [ ] Acceptance Criteria met
+- [ ] Commit messages are meaningful
+- [ ] Manually tested
+- [ ] Self-reviewed your code
+- [ ] Code reviewed from two other developers
+- [ ] If your pull request depends on any other, please link them in the description
+- [ ] main branch is passing/green on CI
```

Comment:
Not possible to do this before the PR is merged
```suggestion
```

Reply:
Ah by that I mean "check that the main branch on our CI is passing before merging", as we don't want to merge new code onto broken code; the code in "master" should be fixed before we merge more code in

Reply:
oh, that makes more sense, I misread it

Reply:
Could help to make that clarification
```suggestion
- [ ] main branch is passing/green on CI, so we know if this PR breaks the build
```

Reply:
I can rename this as `Check main branch is passing/green on CI, to prevent creating new builds after a previously broken build`